Projects/Contracts/Sources/Service/DemoImplemenmtation/ContractsClientDemo.swift

```diff
@@ -27,7 +27,9 @@ public class FetchContractsClientDemo: FetchContractsClient {
                 ),
             ],
             documents: [.init(displayName: "Display name", url: "https://www.hedvig.com", type: .generalTerms)],
-            displayName: "Home Insurance Rent"
+            displayName: "Home Insurance Rent",
+            displayNameTier: "display name tier",
```

Comment:
Can we put some proper value here? Its used for demo mode and will present it when you enter the demo mode.